strongbox-security/strongbox-user-management/src/main/java/org/carlspring/strongbox/users/security/AuthoritiesProvider.java

```diff
@@ -39,88 +32,32 @@
-                                                            new SimpleGrantedAuthority(privilegeName.toUpperCase())));
-                                    }
-                                });
-
-        try
-        {
-            Roles configuredRole = Roles.valueOf(roleName);
-            authorities.addAll(configuredRole.getPrivileges());
-        }
-        catch (IllegalArgumentException e)
-        {
-            logger.warn("Unable to find predefined role by name " + roleName);
-        }
-
-        return authorities;
-    }
-
-    private void populate(Set<GrantedAuthority> fullAuthorities,
-                          Set<Role> configuredRoles)
-    {
-        AuthorizationConfig authorizationConfig = authorizationConfigService.get();
-
-        authorizationConfig.getRoles()
-                           .forEach(
-                                   role -> role.getPrivileges()
-                                               .forEach(
-                                                       privilegeName -> fullAuthorities.add(
-                                                               new SimpleGrantedAuthority(privilegeName.toUpperCase())
-                                                       )
-                                               )
-                           );
-
-        configuredRoles.addAll(authorizationConfig.getRoles());
-    }
-
     public Set<Role> getAssignableRoles()
     {
-        ImmutableSet.Builder builder = ImmutableSet.<Role>builder()
-                                               .addAll(authorizationConfigService.get()
-                                                                                 .getRoles())
-                                               .addAll(Arrays.stream(Roles.values())
-                                                             .map(r -> new RoleDto(r.name(), r.getDescription()))
-                                                             .map(Role::new)
-                                                             .collect(Collectors.toSet()));
-        return builder.build();
+        return authorizationConfigService.get().getRoles();
+    }
+    
+    public RoleData getRuntimeRole(String name) {
+        Role role = authorizationConfigService.get()
+                                         .getRoles()
+                                         .stream()
+                                         .filter(r -> r.getName().equals(name))
+                                         .findFirst()
+                                         .orElseThrow(() -> new IllegalArgumentException(name));
+        
+        if (SystemRole.ADMIN.name().equals(name)) {
+            RuntimeRole adminRole = new RuntimeRole(role, (a) -> new AdminAccessModel(a));
+            return new RuntimeRole(adminRole, (a) -> new AuthorizedAccessModel(a));
+        } else if (SystemRole.ANONYMOUS.name().equals(name)) {
```

Comment:
Cosmetic: Please, change this in accordance with the coding convention:
```
} else if (SystemRole.ANONYMOUS.name().equals(name)) {
```
to:
```
}
else if (SystemRole.ANONYMOUS.name().equals(name))
{
```

Reply:
fixed